tv_apps/smart-system/index.html

```diff
@@ -35,6 +35,7 @@
     <script defer src="shared/js/font_size_utils.js"></script>
     <!-- XXX can be removed while gecko support navigator.mozHour12 API -->
     <script defer src="shared/js/date_time_helper.js"></script>
+    <script defer src="tv_shared/js/simple_key_navigation.js"></script>
```

Comment:
Because system app uses a lot of files shared and tv_shared, we will put these kinds of file at the top of who uses it. Once two modules use it, we will move it to here.